pkg/cmd/configure.go

```diff
@@ -166,25 +164,15 @@ doppler configure set key=123 otherkey=456`,
 		options := map[string]string{}
 
 		if len(args) == 1 && !strings.Contains(args[0], "=") {
-			var input []string
-			scanner := bufio.NewScanner(os.Stdin)
-			// read input from stdin
-			for {
-				if ok := scanner.Scan(); !ok {
-					if e := scanner.Err(); e != nil {
-						utils.HandleError(e, "Unable to read input from stdin")
-					}
-
-					break
-				}
-
-				s := scanner.Text()
-				input = append(input, s)
+			value, ok, err := utils.GetStdIn()
```

Comment:
When the var is named `ok`, this util's signature feels funky; the presence of `err` seems like it should indicate whether or not the read was "ok". However, if this were named "hasData", I think it would make sense. Thoughts on renaming the var here (and at the other callsite)? Alternatives include:

- Changing the signature to `(string, error)` and returning `"", errors.New("Unable to read input from stdin")` directly from `GetStdIn` if there is no data
- Changing the signature to `(*string, error)` and returning `nil, nil` if there is no data

Reply:
I liked your last suggestion of changing the return signature to `(*string, error)` as it seemed the least ambiguous. Change has been pushed!